utils/s2n_random.c

```diff
@@ -180,32 +180,326 @@ S2N_RESULT s2n_get_urandom_data(struct s2n_blob *blob)
  */
 S2N_RESULT s2n_public_random(int64_t bound, uint64_t *output)
 {
-    uint64_t r;
+    uint64_t x;
 
     ENSURE_GT(bound, 0);
```

Comment:
NitPick: I'd move these after the block-comment so as not to break up the actual code.